### PR TITLE
Update UG with Command Benefits

### PR DESCRIPTION
**Pull request by bchenghi**
- Update commands with benefits.
- Remove bolding for "About this document" hyperlinks, since all hyperlinks are not bolded.
- Each command has command description, then format, then bullet points for more details. (Changed for sorting commands)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ The `delete` command deletes a specified food item from your food inventory.
 
 ## Editing a food item
 
-The edit command edits the details of an existing food item in your food inventory.
+The edit command edits the details of an existing food item in your food inventory. If an entry was incorrect, you can easily edit the entry, without deleting and re-adding the food item.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Perhaps it should be "If an entry is incorrect, ..."?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ After entering this command, you should see a pop-up window similar to the one s
 
 ## Viewing expired food items
 
-When you launch the app, it checks for any expired food present in your food inventory.
+When you launch the app, it checks for any expired food present in your food inventory. This lets you easily see what foods have expired.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Maybe can change "food" to "food item" to make it consistent with the rest of the UG? Same for "foods" as well.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -265,33 +265,33 @@ The `sortdesc` command sorts the list of food items by description.
 
 ## Sort food items by expiry date
 
-The `sortexpiry` command sorts the list of food items by expiry date from oldest to newest. 
+The `sortexpiry` command sorts the list of food items by expiry date from oldest to newest. With this, you can easily tell which foods are expiring first.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Same as above, maybe can consider using "food items" instead?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -265,33 +265,33 @@ The `sortdesc` command sorts the list of food items by description.
...
-**Format:** `sortexpiry`
-
 ![sorting by expiry date from oldest to newest](images/SortExpiry1.3.png)
 
 <div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html">^ Back to top</a></div>
 
 ## Sort food items by priority
 
-The `sortpriority` command sorts the list of food items by priority from high to low. 
+The `sortpriority` command sorts the list of food items by priority from high to low. With this, you can easily tell which foods have higher priority.
```
Comment (jonahtanjz) [inline docs/UserGuide.md]: Same for "foods" here.

**Review by jonahtanjz — COMMENTED** [review]
Some minor suggestions. Else LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (bchenghi) [reply docs/UserGuide.md]: Changed. Thanks!

File: docs/UserGuide.md (hunk 2 above)
Comment (bchenghi) [reply docs/UserGuide.md]: Updated, thanks.

File: docs/UserGuide.md (hunk 3 above)
Comment (bchenghi) [reply docs/UserGuide.md]: Updated, thanks.

File: docs/UserGuide.md (hunk 4 above)
Comment (bchenghi) [reply docs/UserGuide.md]: Changed, thanks!

**Review by jonahtanjz — APPROVED** [review]
LGTM!

**Pull request merged**